src/pretix/plugins/banktransfer/templates/pretixplugins/banktransfer/pending.html

```diff
@@ -103,3 +104,20 @@
 {% if swiss_qrbill %}
 <link rel="stylesheet" href="{% static "pretixplugins/banktransfer/swisscross.css" %}">
 {% endif %}
+
+{% if has_invoices %}
```

Comment:
Should we maybe also only show this if the `invoice_email` setting on the payment provider has been set? At least to me it makes sense if the email option during checkout and the email option after checkout are controlled by the same setting.

Reply:
Sounds reasonable, especially as the config options for the mail text are only visible if invoice_email is set.